### ISSUE TITLE
this repo needs to be synced with main spack repo

### ISSUE THREAD

**Issue by edwardhartnett** [JCSDA/spack#209]
### Steps to reproduce

It does us no good to have our own version of spack. We need to expose it to the community and also let the community help us.

So we need to merge our changes to the main spack repo. We need users to be able to build from the main spack repo, and have it work the same as for us.

It's not clear that the NCO will accept updates from our fork of the main spack repo. They will accept updates on the main spack repo. That means the NCEPLIBS team will not be updating the fork of spack, we will instead be updating the main spack repo when we do a release.

THis spack fork, if it continues to be used, must fetch those changes from the main spack repo. Yet we do not seem to be updating this fork from the main spack repo  - we need to.

### Error message

v

### Information on your system

f

### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have searched the issues of this repo and believe this is not a duplicate
- [X] I have run the failing commands in debug mode and reported the output

**Comment by climbfuji** [comment]
This is a duplicate of #28.

> It's not clear that the NCO will accept updates from our fork of the main spack repo. They will accept updates on the main spack repo. That means the NCEPLIBS team will not be updating the fork of spack, we will instead be updating the main spack repo when we do a release.

Let me ask the reverse question. Is spack going to accept the changes that NCO requires us to put into a number of packages so that they can be installed on FISMA HIGH systems?

**Comment by climbfuji** [comment]
I do agree with you of course that we need to keep the differences minimal. To get things started, I am first going to update our fork from the authoritative repo, so that we can send our changes back cleanly. PR to come

**Comment by edwardhartnett** [comment]
Excellent. Once that is done I will see if there are any differences in the NCEPLIBS library package files that can be pushed up to the main repo.

**Comment by climbfuji** [comment]
@edwardhartnett We just merged the update from the authoritative repo as of Dec 22. Since both repos are marching ahead all the time, you need to compare the following hashes to get a good idea of what is different:
noaa-emc fork, jcsda_emc_spack_stack branch: 3a152138df8f6db58019b1b8d19b75a0bbbd0c23
spack repo, develop branch: 38c1639c9cb073c3cbaced8a0c519cda7e28b453

**Comment by climbfuji** [comment]
We are making progress on this.

**Comment by climbfuji** [comment]
CLosing this, since it is a duplicate of #28
